src/jit/gentree.h

```diff
@@ -1401,6 +1401,20 @@ struct GenTree
         return OperIsShiftOrRotate(OperGet());
     }
 
+    static bool OperIsMul(genTreeOps gtOper)
+    {
+        return (gtOper == GT_MUL) || (gtOper == GT_MULHI)
+#if !defined(_TARGET_64BIT_) && !defined(LEGACY_BACKEND)
```

Comment:
This will be a conflict with @BruceForstall change to remove LEGACY_BACKEND

Reply:
Yes, there are probably more conflicts as well; that's how it goes ;-)